lib/morse_translator.rb

```diff
@@ -40,4 +40,22 @@ def initialize
                     " " => " "}
   end
 
+  def translate(english)
+    english.downcase.split("").map { |letter| @dictionary[letter]}.join
+  end
+
+  def morse_to_eng(morse_message)
+    english_array = morse_message.split("  ").map do |morse_word|
+      morse_word_to_eng(morse_word)
+    end
+    english_array.flatten.join.strip
+  end
+
+  def morse_word_to_eng(morse_word)
+    english_word = morse_word.split.map do |morse_character|
+      @dictionary.find{|key, value| value == morse_character}[0]
+    end
+    english_word << " "
```

Comment:
Because this method is named morse_word_to_eng, I would expect it to return a string representing the word instead of an array. Doing this would also allow you to use `join(" ")` up above instead of `.join.strip` which wouldn't work in all scenarios i.e. if you wanted the sentence to end in a space

Reply:
I need to practice breaking the habit of packing as much stuff into a line as possible, and avoid "cleverness" where I can just make it more straightforward. Looks like I also need to pump the breaks and scrutinize my code more closely before submitting it. I left out some pretty obvious stuff because I didn't do a line-by-line review when I thought I was done.

Thanks for the feedback! I'll be sure to pay closer attention next time around.